src/docs/introduction/versioning.adoc

```diff
@@ -11,15 +11,16 @@ If necessary a 4th level point release number will be used for successive change
 
 |===
 s| Plugin Version    s| Grails                s| Elasticsearch
+s| 1.4.1           s| 3.1.x                 s| 5.x (tested with 5.4.1)
```

Comment:
No according to the docs http://noamt.github.io/elasticsearch-grails-plugin/docs/index.html#plugin-versioning

1.4.x | 3.1.x | (hypothetical) 5.0.x

"where __there isn’t really a 1-to-1 plugin version to grails or es version, but we just increase our major or minor version by one__, whenever there are breaking changes on either Grails or ES"